src/routes/jobInvitesRoutes.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
```

Comment:
There should be nothing like `jobinvitesRoutes.js` deletthe file and place your endpoint inside `routes.js` read the comments there so you'll know where to place it